lib/ae_mdw/db/util.ex

```diff
@@ -256,10 +256,21 @@ defmodule AeMdw.Db.Util do
       node_progress: node_progress,
       mdw_version: AeMdw.MixProject.project()[:version],
       mdw_height: mdw_height,
-      mdw_tx_index: AeMdw.Db.Util.last_txi(),
+      mdw_tx_index: mdw_tx_index,
       mdw_synced: node_height == (mdw_height + 1), # MDW is always 1 generation behind
       mdw_syncing: mdw_syncing?
     }
   end
 
+  defp safe_mdw_tx_index_and_height do
+    try do
+      mdw_tx_index = last_txi()
+      {mdw_height, _} = read_tx!(mdw_tx_index) |> Model.tx(:block_index)
+      {mdw_tx_index, mdw_height}
+    rescue
+      e in RuntimeError ->
```

Comment:
better use just `_` to catch all, without var name as that would emit unused warning...